constants.ts

```diff
@@ -1,5 +1,7 @@
 export const appName = "Saleor Avatax";
 
+export const DEFAULT_TAX_CODE = "O9999999"
```

Comment:
What is this tax code, where does it come from? Could you add a link to docs that describe it?